server/services/tests/conversationServices.test.js

```diff
@@ -0,0 +1,54 @@
+let conversationServicesFactory = require('../conversationServices');
+const chai = require("chai");
+const expect = chai.expect;
+const sinon = require("sinon");
+
+describe("conversationServices", function() {
```

Comment:
you ignored a lot of use cases in createConversation method and getConversationMetadata